bpf/Makefile.bpf

```diff
@@ -3,7 +3,8 @@ FLAGS := -I$(ROOT_DIR)/bpf/include -I$(ROOT_DIR)/bpf -D__NR_CPUS__=$(shell nproc
 CLANG_FLAGS := ${FLAGS} -target bpf -emit-llvm
 # eBPF verifier enforces unaligned access checks where necessary, so don't
 # let clang complain too early.
-CLANG_FLAGS += -Wall -Wextra -Werror -Wno-address-of-packed-member -Wno-unknown-warning-option
+CLANG_FLAGS += -Wall -Wextra -Werror
+CLANG_FLAGS += -Wno-address-of-packed-member -Wno-unknown-warning-option -Wno-gnu-variable-sized-type-not-at-end
```

Comment:
nice. I guess Clang must just like to complain on this one, but still generate the right code (zero size for VLAIS). Maybe trying to catch potential bugs if developers use it wrong.

Reply:
I'm not sure what is the reason for it exactly... not liking GNU extensions? ;) There is some related discussion here as well: https://lkml.org/lkml/2017/10/17/66 .